src/Kinepolis/AuthenticatedKinepolisClient.php

```diff
@@ -72,6 +73,33 @@ public function getMovieDetail(string $token, int $mid): array
         return $contents['movies'][0];
     }
 
+    public function getImage(string $token, string $path): UploadedFile
+    {
+        $lastSlashPosition = strrpos($path, '/');
+
+        $fileName = substr($path, $lastSlashPosition + 1);
```

Comment:
I would move this a separate function with a function name that explains what / why you are doing this.